docs/add-ons/overview.mdx

```diff
@@ -1,67 +1,162 @@
-# Overview
+# Introduction to Add-ons
```

Comment:
I think we should call it "Addons" over "Add-ons", we would like to unify the name along with our APIs, since there have been a lot of mixed styles (Addon, Add-on, AddOn, etc.) lately. This applies to all occurrences.